content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -151,7 +151,10 @@ stages:
 by the user with the `--name` (`-n`) option of `dvc run`. Each stage can contain
 the possible following fields:
 
-- `cmd` (always present): Executable command defined in this stage
+- `cmd` (always present): Executable commands defined in this stage. This could be
+  either a single command or a list of commands that should be executed sequentially.
+  The failure of one of these commandes will result in the failure of the stage.
+  Note that each command is passed as is to the shell.
```

Comment:
Cool.

> The failure of one of these commandes will result in the failure of the stage.

Typo: commandes 

How does that work exactly? I assume it stops and doesn't execute the remaining commands but we should be explicit. Also, what happens to the outputs of previous successful commands?

Reply:
p.s. Feel free to use multiple paragraphs or a block quote (note) inside this bullet to make it easier to read since it's getting long.